src/torchcodec/_core/Transform.cpp

```diff
@@ -42,7 +42,7 @@ int toSwsInterpolation(ResizeTransform::InterpolationMode mode) {
 std::string ResizeTransform::getFilterGraphCpu() const {
   return "scale=" + std::to_string(outputDims_.width) + ":" +
       std::to_string(outputDims_.height) +
-      ":sws_flags=" + toFilterGraphInterpolation(interpolationMode_);
+      ":flags=" + toFilterGraphInterpolation(interpolationMode_);
```

Comment:
From the [FFmpeg docs](https://ffmpeg.org/ffmpeg-filters.html#Filtergraph-syntax-1):

> Libavfilter will automatically insert [`scale`](https://ffmpeg.org/ffmpeg-filters.html#scale) filters where format conversion is required. It is possible to specify swscale flags for those automatically inserted scalers by prepending `sws_flags=flags;` to the filtergraph description.

Whereas `flags` is the [specific parameter to `scale`](https://ffmpeg.org/ffmpeg-filters.html#Options-2). They end up being semantically equivalent, but it's more clear to use the scale option here.